src/components/ScholarshipForm.jsx

```diff
@@ -60,14 +61,24 @@ const useStyles = makeStyles((theme) => ({
 function ScholarshipForm({ scholarship, submitFn, onSubmitError }) {
   const classes = useStyles();
   const [activeStep, setActiveStep] = useState(0);
-  const [hasReqs, setHasReqs] = useState(false);
+  const [noReqsChecked, setNoReqsChecked] = useState(false);
+
+  const [noReqsHelperText, setNoReqsHelperText] = useState('');
 
   const formik = useFormik({
     initialValues: scholarship.data,
     validationSchema,
     onSubmit: (values, { setSubmitting, resetForm }) => {
       setSubmitting(true);
       scholarship.data = { ...values };
+
+      if (noReqsChecked) {
+        Object.keys(scholarship.data.requirements).forEach(
+          (k) => (scholarship.data.requirements[k] = [])
+        );
+        scholarship.data.requirements.gpa = 0;
```

Comment:
If the user chooses some requirements and then checks the 'No requirements checkbox', this will 'save' their choices, but not submit them to the scholarship.

Another thing we could do is clear any selections they made before checking the checkbox. Not sure what route we wanna take.